ci/test/00_setup_env_native_previous_releases.sh

```diff
@@ -16,5 +16,6 @@ export RUN_UNIT_TESTS_SEQUENTIAL="true"
 export RUN_UNIT_TESTS="false"
 export GOAL="install"
 export DOWNLOAD_PREVIOUS_RELEASES="true"
-export BITCOIN_CONFIG="--enable-zmq --with-gui=qt5 --enable-reduce-exports --enable-debug \
-CFLAGS=\"-g0 -O2 -funsigned-char\" CPPFLAGS='-DBOOST_MULTI_INDEX_ENABLE_SAFE_MODE' CXXFLAGS=\"-g0 -O2 -funsigned-char\""
+export BITCOIN_CONFIG="-DWITH_ZMQ=ON -DBUILD_GUI=ON -DREDUCE_EXPORTS=ON -DCMAKE_BUILD_TYPE=Debug \
+-DCMAKE_C_FLAGS_DEBUG='-g0 -O2 -funsigned-char' \
```

Comment:
Shouldn't the `funsigned-char` here be `-DCMAKE_C[XX]_FLAGS` since it's not part of the optims?

Reply:
In https://github.com/hebasto/bitcoin/pull/142#discussion_r1554395773, @fanquake expressed the opposite opinion :) 

The result will be the same for both approaches. The one you ask about is just a bit more verbose.

Do you want me to apply it?

Reply:
Hah.

I suppose it's up to @fanquake. Imo using both vars is more correct, but I don't feel too strongly.

Reply:
> I suppose it's up to @fanquake. Imo using both vars is more correct, but I don't feel too strongly.

Now that we've (re-)changed approach, we can split them back out into the all the different vars.

Reply:
Thanks! done in the recent push.